language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/TransactionClientImpl.java

```diff
@@ -25,110 +27,190 @@
     public TransactionClientImpl(String ledgerId, Channel channel, ExecutionSequencerFactory sequencerFactory) {
```

Comment:
Is it a common use-case to use different access tokens per calls on this interface? If not, it would be simpler to pass the access token here once.

Reply:
This will come in the following PR that closes the ticket. You need to have the possibility of passing tokens on a per-call basis to be able to work with possibly expiring tokens.